<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cipher</class>
 <widget class="QMainWindow" name="Cipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 240, 239);
border-color: rgb(118, 0, 59);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="knopka_zashifrovat">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>Зашифровать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knopka_deshifrovat">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>Дешифровать</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tyt_vivodit_soobshenie">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>320</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(145, 145, 145);
background-color: rgb(239, 239, 239);</string>
    </property>
   </widget>
   <widget class="QLabel" name="soobshenie_vvod">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Сообщение для зашифрования/дешифрования</string>
    </property>
   </widget>
   <widget class="QLabel" name="soobshenie_vivod">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифрованное/дешифрованное сообщениие</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vvedite_keyword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ключ-слово:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_viberite_language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Программа поддерживает только русский язык.</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tyt_vvodit_keyword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 239, 239);</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tyt_vvodit_soobshenie">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>320</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 239, 239);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
